custom_components/huesensor/binary_sensor.py

```diff
@@ -33,12 +34,25 @@
         "reachable",
         "sensitivity",
         "threshold_dark",
-        "threshold_offset",
+        "threshold_offset"
+    ],
+    "PHD": [
+        "on",
+        "configured",
+        "sunrise_offset",
+        "sunset_offset",
+        "name",
+        "type",
+        "modelid",
+        "swversion",
+        "daylight",
+        "last_updated",
     ]
+
 }
 
 
-def parse_hue_api_response(sensors):
+def parse_hue_api_response(bridgeid, sensors):
```

Comment:
Why are you passing `bridgeid`?

Reply:
To build unique names for the sensor when multiple bridges are involved.
There is not "unique_id" for that sensor...

Reply:
That would also apply to remotes etc then. This should be applied consistently

Reply:
Not sure what you mean.
All the "actual devices" have their own unique_id, assumed to be unique across bridges:
```json
"uniqueid": "00:17:88:01:10:4f:75:9b-0b"
```

Reply:
Added to the above , the daylight sensor is a per hub sensor, and not a Hue.
 accessory 
We need the hub id because people have multiple hub setups

Reply:
But hey, this is more an intellectual exercise than anything else.
This sensor is 1:1 with the sun sensor of HA, and I don't see a use case where we would use this sensor rather than the HA builtin.

I'd actually be rather against merging this, as it would add another entity that nobody would ever use, imho ;)

Reply:
I am inclined to agree, unless there is a very good reason to include it..?

Reply:
well, I am using it right now, hence my efforts to use it.

not sure if this is 1:1 with core sun component, it is dependent on the user config in the hue hub and taken into account in the Hue hub logic for home/way, day/night schedules.

I find the use of the core Hue settings in HA to be very useful (also use the schedules, sensitivities etc, and can set these from within HA)

this 1 extra sensor doesn't do any harm in the HA ecosystem, and would be a nice addition to it on the contrary imho.

Reply:
OK I do see the benefit. However I do want to find a better way to handle the multiple hue hub issue. For example, the hue hub id could be an attribute on all the sensors on that hub

Reply:
> well, I am using it right now

Well sure, but the question is, in the context of HA, why you're using that one rather than "sun"?

There is a bunch of additional attributes in "sun" (https://www.home-assistant.io/integrations/sun/) and it's handled specifically as far as automations are involved, ie you can specify offsets in the trigger (https://www.home-assistant.io/docs/automation/trigger/#sun-trigger) while the hue offsets can only be altered with the hue app.

The only config needed is your geographic location in HA.

Reply:
tbh, since the introduction of native Hue sensors in HA, one could ask this question Why, on most of the entities this CC creates...

the answer is always: because it is better, more flexible, quicker etc etc.
In this particular case: I mimic the Hue setup in my HA config. The one sensor missing was the daylight setting in the Hub.

This will allow for further logic in HA to use in home/away schedules, and the options the App offers (Hue is expanding that at the moment, given the extra rules being created)

I am aware of what we can do in the HA ecosystem, like offsets in triggers etc, but you might also know that exactly that subject is cause for great confusion in the community.

Having a clear daylight sensor being on/off following Hue rules is very very adequate and easy to use. No more confusions which (light, day, dark, above_horizon,offset)sensor to use to mimic Hue behavior. Simply use hue_daylight.

Reply:
OK my decision is that at this time there is not a strong case for inclusion - each line of code adds a burden of maintainence and possibility for bugs. @Mariusthvdb please create a feature request (citing this PR) highlighting why this is substantially different from the sun sensor. If there are several people keen then I will consider again. Thanks

Reply:
consider that done: thanks for the opportunity!
https://github.com/robmarkcole/Hue-sensors-HASS/issues/212